### PR TITLE
feature/OP-457

### PR DESCRIPTION
**Pull request by estevez90**
Edit email notifications

### CODE REVIEW

File: public_records_portal/notifications.py [hunk 1]
```diff
@@ -125,6 +126,8 @@ def generate_prr_emails(
     #     template = 'system_email_' + notification_type[-2:] + '.html'
```
Comment (joelbcastillo) [inline public_records_portal/notifications.py]: Is this commented out code needed?

Comment (estevez90) [inline public_records_portal/notifications.py]: I've removed the commented out code

**Comment by joelbcastillo** [comment]
The confirmation box should look like this: [Example Confirmation Box](https://www.dropbox.com/s/uwa58dhoppt813t/Screenshot%202016-04-05%2016.09.45.png?dl=0)

This will allow the FOIL Officers to view the entire email before sending it out to the requester. 

Additionally, can we increase the size of the modal so that the full email and edit box are viewable without scrolling in the modal.

**Comment by joelbcastillo** [comment]
Also, if I hit cancel after editing the text, and then select a new option that sends out an email, the old confirmation box (including the same question).

**Comment by joelbcastillo** [comment]
The email that is generated when editing the notification is not formatted correctly: 

screenshot 2016-04-05 16 20 20

**Comment by joelbcastillo** [comment]
When uploading a document listed as "Private" there should not be an email notification generated.

**Comment by joelbcastillo** [comment]
**This issue still exists:**

Also, if I hit cancel after editing the text, and then select a new option that sends out an email, the old confirmation box (including the same question).

**Comment by joelbcastillo** [comment]
When closing a request, request is closed, but application generates a 404 error page. Also, emails are not sent when closing a request
screenshot 2016-04-07 19 31 14

**Comment by aot221** [comment]
Ckeditor isn't showing up on my computer.

screen shot 2016-04-08 at 3 47 58 pm

Also the Ckeditor pops up when you switch the privacy setting from 'private' to 'release and public'.

**Comment by joelbcastillo** [comment]
Merged manually

**Pull request closed without merging**